src/drawing/sprite.c

```diff
@@ -360,10 +360,10 @@ void FASTCALL gfx_draw_sprite_software(rct_drawpixelinfo *dpi, int image_id, int
 * x (cx)
 * y (dx)
 */
-void FASTCALL gfx_draw_sprite_palette_set_software(rct_drawpixelinfo *dpi, int image_id, int x, int y, uint8* palette_pointer, uint8* unknown_pointer)
+void FASTCALL gfx_draw_sprite_palette_set_software(rct_drawpixelinfo *dpi, sint32 image_id, sint32 x, sint32 y, uint8* palette_pointer, uint8* unknown_pointer)
```

Comment:
No idea, it's been that way since https://github.com/OpenRCT2/OpenRCT2/commit/b828f678 apparently.